include/term-slug.php

```diff
@@ -155,19 +160,34 @@ private function maybe_get_parent_suffix() {
 	 *
 	 * @since 3.7
 	 *
-	 * @param string $separator The separator for the slug suffix, or empty.
+	 * @param string $separator   The separator for the slug suffix, or empty.
+	 * @param bool   $suffix_slug Should force suffix slug, or not, default false.
 	 * @return string
 	 */
-	public function get( string $separator = '' ): string {
+	public function get( string $separator = '', bool $suffix_slug = false ): string {
```

Comment:
Yes but actually it is to handle another case.

When we're modifying an existing term in `PLL_Share_Term_Slug` we have to find a way of forcing the suffixed slug, because if we already have a `$term_id ` [here](https://github.com/polylang/polylang/blob/17f1540d3976556bdf1857105c355b6364cee1a4/include/term-slug.php#L173) we'll return the slug without a suffix.

Reply:
I thought that's what we'd want.
So in Polylang Pro:
- pass `can_add_suffix()`
- get the `term_id` and compare it to the globals
- if the test passes, call `get( self::TERM_SLUG_SEPARATOR )` because at this point we do want to add a suffix and the `term_id` is already guessed and do exists.